x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form.tsx

```diff
@@ -405,16 +402,6 @@ export const RuleForm = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [ruleTypeRegistry, availableRuleTypes, searchText, JSON.stringify(solutionsFilter)]);
 
-  useEffect(() => {
```

Comment:
By deleting that we are messing up the way we set up the consumer for rule who has been created in o11y rule management. At this point we want to use the rule producer as consumer and not use `alerts` to match what the reality is.

Reply:
Same as above